autoscale/supervisor_client.go

```diff
@@ -89,6 +104,7 @@ func UnassignTenant(podIP string, tenantName string, forceShutdown bool) (resp *
 			}
 			Logger.Errorf("[error][SupClient]failed to UnassignTenant, grpc_err: %v  api_err: %v", err1, err2)
```

Comment:
add two type into metric: SupervisorClientUnassignTenantGrpcErrCnt , SupervisorClientUnassignTenantRespErrCnt.
also add err type metric for `assignTenant` and `getCurrentTenent`